<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>class_systemClass</class>
 <widget class="QMainWindow" name="class_systemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>排课系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="set">
          <property name="text">
           <string>学时</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choose">
          <property name="text">
           <string>自动排课</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fallback">
          <property name="text">
           <string>回退</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>选课系统</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="show_dag">
          <property name="text">
           <string>显示课程先后关系</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="change_term">
          <property name="text">
           <string>调整课程</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="class_table">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>8</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>第一学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第二学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第三学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第四学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第五学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第六学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第七学期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第八学期</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="debug">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="class_system.qrc"/>
 </resources>
 <connections/>
</ui>
